roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,13 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: change swift log file permissions
+  file: path={{ item }} mode=0644 owner=swift group=swift state=touch
+  with_items:
+    - /var/log/swift/account.log
+    - /var/log/swift/container.log
+    - /var/log/swift/object.log  
+
```

Comment:
That's a good note, and probably should be done in future techspec PRs (and retroactively in previous changes).